### PR TITLE
Move `autoround` from `generate.py` to `eval.py`

### PR DESCRIPTION
**Pull request by yiliu30**
As `Auto-Round` is an algorithm focused on improving accuracy instead of perf, move it from `generate.py` to `eval.py`.

cc @thuang6

### CODE REVIEW

**Comment by yiliu30** [comment]
Hi @HDCharles @jerryzh168, please have a look, thanks!

**Review by jerryzh168 — APPROVED** [review]

File: torchao/_models/llama/eval_acc.sh
Comment (jerryzh168) [inline torchao/_models/llama/eval_acc.sh:1]: nit: maybe just `evals.sh` following the same naming convention as benchmarks

Comment (yiliu30) [reply torchao/_models/llama/eval_acc.sh:1]: Thanks, updated it.

**Comment by HDCharles** [comment]
I added evaluations.sh into a PR a few hours ago, can you move evals.sh into there?

**Review by HDCharles — APPROVED** [review]
otherwise looks good

**Comment by yiliu30** [comment]
Ready for merge.

**Pull request merged**